samples/touchtest/touchtest.cpp

```diff
@@ -0,0 +1,251 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        touchtest.cpp
+// Purpose:     Multitouch sample
+// Author:      Martin Koegler
+// Modified by:
+// Created:     2015-07-29
+// Copyright:   (c) Martin Koegler
+// Licence:     wxWindows licence
+/////////////////////////////////////////////////////////////////////////////
+
+#include "wx/wx.h"
+#include "wx/dcbuffer.h"
+
+// Define a new application
+class wxMyApp: public wxApp
+{
+public:
+    bool OnInit() override;
+};
+
+wxDECLARE_APP(wxMyApp);
+
+constexpr unsigned int TOUCH_POINTS = 17;
+
+class MyFrame: public wxFrame
+{
+private:
+    struct TouchState
+    {
+        wxTouchSequenceId id;
+        wxPoint2DDouble last;
+        wxPen pen;
+    };
+    TouchState m_TouchPoints[TOUCH_POINTS];
```

Comment:
Minor but

```suggestion
    std::array<TouchState, TOUCH_POINTS> m_TouchPoints;
```

would be a bit nicer.